event_lead_image/tests/test_lead_image.py

```diff
@@ -0,0 +1,41 @@
+from odoo.tests.common import SavepointCase
+import os
+from base64 import b64encode, b64decode
+
+
+class TestLeadEvent(SavepointCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        path = os.path.dirname(__file__) + '/assets/testimage.png'
+        file_ = open(path, 'rb')
+
+        cls.testevent = cls.env['event.event'].create({
+            'name': 'Testevent',
+            'date_begin': '2018-02-13 08:00:00',
+            'date_end': '2018-02-13 10:00:00',
+            'lead_image': b64encode(file_.read())
+        })
+
+    def test_access_normal(self):
+        """Check if we get correct base64 strings back."""
+        self.assertTrue(b64decode(self.testevent.lead_image_big))
+        self.assertTrue(b64decode(self.testevent.lead_image_medium))
+        self.assertTrue(b64decode(self.testevent.lead_image_small))
+
+    def test_access_binsize(self):
+        """Test that we don't fail if we're in bin_size mode."""
+        event = self.env['event.event'].with_context(
+            bin_size=True).browse(self.testevent.id)
```

Comment:
comment: `bin_size` make the fields return images' sizes instead of image content